modules/serverless-subscription-event-delivery-config.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// serverless/event_workflows/serverless-event-delivery.adoc
+
+[id="serverless-subscription-event-delivery-config_{context}"]
+= Configuring event delivery failure parameters using subscriptions
+
+Developers can configure event delivery parameters for individual subscriptions by modifying the `delivery` settings for a `Subscription` object.
+
+.Example subscription YAML
+[source,yaml]
+----
+apiVersion: messaging.knative.dev/v1beta1
+kind: Subscription
+metadata:
+  name: <subscription_name>
+  namespace: <subscription_namespace>
+spec:
+  delivery:
+    deadLetterSink: <1>
+      ref:
+        apiVersion: serving.knative.dev/v1
+        kind: Service
+        name: <sink_name>
+    backoffDelay: <duration> <2>
+    backoffPolicy: <policy_type> <3>
+    retry: <integer> <4>
+----
+<1> Configuration settings to enable using a dead letter sink. This tells the subscription what happens to events that cannot be delivered to the subscriber.
++
+When this is configured, events that failed to be delivered are sent to the dead letter sink destination. The destination can be a Knative service or a URI.
+<2> You can set the `backoffDelay` delivery parameter to specify the time delay before an event delivery retry is attempted after a failure. The duration of the `backoffDelay` parameter is specified using the https://en.wikipedia.org/wiki/ISO_8601#Durations[ISO 8601] format. For example, `PT1S` specifies a 1 second delay.
```

Comment:
`backoffDelay` and `backoffPolicy` are not described in the section above about common eventing delivery options, but I suspect that they are also available for brokers ?

Reply:
This module is included in the assembly immediately below https://deploy-preview-30282--osdocs.netlify.app/openshift-enterprise/latest/serverless/event_workflows/serverless-event-delivery.html#serverless-event-delivery-parameters - again please look at the actual preview docs when reviewing

Reply:
Also, to set this for brokers as I've mentioned previously will be a future use case once this is available for triggers